api-server/src/common/models/user.json

```diff
@@ -276,9 +276,7 @@
       "default": {}
     },
     "donationEmails": {
-      "type": [
-        "string"
-      ]
+      "type": ["string"]
```

Comment:
Is this just a cosmetic change, or did we lose some changes during the rebasing?

Reply:
This is a prettier change. I got the same change while working on #42663

Reply:
Awesome, thanks for confirming.